network/network_linux.go

```diff
@@ -476,6 +476,14 @@ func (nm *networkManager) connectExternalInterface(extIf *externalInterface, nwI
 		return err
 	}
 
+	//fetch ovs port number and add it to nwInfo struct
+	extIf.OVSSwitchPort, err = networkClient.GetOVSPort(extIf)
+	if err != nil {
+		log.Printf("[ovs] Get ofport failed with error %v", err)
+		return err
+	}
+	nwInfo.OVSSwitchPort = extIf.OVSSwitchPort
```

Comment:
since we directly set to extIf which is part if nw structure...no need to copy this in nwInfo